packages/front-end/components/Metrics/DateGraph.tsx

```diff
@@ -80,81 +61,51 @@ type ExperimentDisplayData = {
 const DateGraph: FC<{
   type: MetricType;
   groupby?: "day" | "week";
+  method?: "avg" | "sum";
   dates: Datapoint[];
   showStdDev?: boolean;
   experiments?: Partial<ExperimentInterfaceStringDates>[];
   height?: number;
 }> = ({
   type,
-  dates,
   groupby = "day",
+  method = "avg",
+  dates,
   showStdDev = true,
   experiments = [],
   height = 220,
 }) => {
   const data = useMemo(
     () =>
-      dates
-        .reduce(
-          (
-            dates: {
-              key: number;
-              total: number;
-              stddev: number;
-              count: number;
-            }[],
-            { d, v, s, c }
-          ) => {
-            const key = (groupby === "day"
-              ? getValidDate(d)
-              : setDay(getValidDate(d), 0)
-            ).getTime();
+      dates.map((row, i) => {
+        const key = getValidDate(row.d).getTime();
+        let value = method === "avg" ? row.v : row.v * row.c;
+        let stddev = method === "avg" ? row.s : 0;
+        const count = row.c || 1;
 
-            const count = c || 1;
-            const total = v * count;
-            const stddev = s;
+        if (groupby === "week") {
+          // get 7 day average (or < 7 days if at beginning of data)
```

Comment:
I feel like we should just drop that data so all data in the graph has the same data generating process, rather than having this "additional variability" that will show up at the beginning of the time series.